scripts/loc_simple.sh

```
@@ -113,14 +302,67 @@ count_functional_area() {
+
+        local ext="${FILE_EXTS[$idx]}"
+        if [[ -z ${allowed_exts["$ext"]+x} ]]; then
+            continue
+        fi
 
-    if [[ $total -gt 0 ]]; then
-        printf "  %-20s: %6d lines (py:%5d js:%4d html:%4d)\n" "$name" "$total" "$py_count" "$js_count" "$html_count"
+        local path="${FILE_PATHS[$idx]}"
+        if [[ -n "$scope_glob" ]]; then
+            case "$path" in
+                $scope_glob) ;;
```

Shellcheck (0.11.0)

[warning] Quote expansions in case patterns to match literally rather than as a glob.

(SC2254)